src/api/cow/cow.spec.ts

```diff
@@ -159,30 +158,34 @@ test('Valid: Get an order ', async () => {
 })
 
 test('Invalid: Get an order ', async () => {
-  fetchMock.mockResponseOnce(
+  // given
```

Comment:
nit: normally we would write a basic sentence in the given.

i.e ''GIVEN: The API returns a 404 Not Found for any order"

Reply:
I rather not.
The code should be self explanatory. Writing a sentence here would be redundant and yet another thing to maintain.
I only use the comments as a divider between sections.